bot.py

```diff
@@ -264,6 +264,28 @@ async def show_final_message_in_embed(ctx):
     embed_new.add_field(name="Final message", value=f"{final_message}", inline=True)
     await ctx.send(embed=embed_new)
 """
+"""<Member id=306869504237895681 name='Uniqe' discriminator='4522' 
+    bot=False nick=None guild=<Guild id=738142745780027484 name='Warzone chłopaczki' 
+    shard_id=0 chunked=True member_count=14>>
+"""
```

Comment:
Co to za kawałek komentarza?

Reply:
powinienem był to usunąć. Potrzebowałem przez chwilę, ale nie powinno tego być